app/views/create-persistent-volume-claim.html

```diff
@@ -0,0 +1,55 @@
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle">
+    <!-- Middle section -->
+    <div class="middle-section surface-shaded">
+      <div class="middle-container has-scroll">
+        <div class="middle-content">
+          <div class="container surface-shaded">
+            <div class="col-md-12">
+              <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+              <alerts alerts="alerts"></alerts>
+
+              <div class="row">
+                <div class="create-route-icon col-md-2 gutter-top hidden-sm hidden-xs">
+                  <span class="pficon pficon-storage-domain icon-xl"></span>
```

Comment:
To be consistent with Attach Storage it should be 'fa fa-hdd-o icon-xl'

On Wed, Jul 20, 2016 at 9:38 AM, Sam Padgett <notifications@github.com>
wrote:

> In app/views/create-persistent-volume-claim.html
> <https://github.com/openshift/origin-web-console/pull/189#discussion_r71525252>
> :
>
> > +  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
> > +    <navbar-utility-mobile></navbar-utility-mobile>
> > +  </div>
> > +  <div class="middle">
> > +    <!-- Middle section -->
> > +    <div class="middle-section surface-shaded">
> > +      <div class="middle-container has-scroll">
> > +        <div class="middle-content">
> > +          <div class="container surface-shaded">
> > +            <div class="col-md-12">
> > +              <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
> > +              <alerts alerts="alerts"></alerts>
> > +
> > +              <div class="row">
> > +                <div class="create-route-icon col-md-2 gutter-top hidden-sm hidden-xs">
> > +                  <span class="pficon pficon-storage-domain icon-xl"></span>
>
> Hm, is this the right icon for PVC? @jwforres
> <https://github.com/jwforres>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/189/files/7e28b3e1330c953d9f77af4dc61380252fe95427#r71525252>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABZk7WlgjhPJZ_WAWRg6tSC81L28KTaIks5qXiTTgaJpZM4JPOcR>
> .
>